### ISSUE TITLE
`prePro` issue

### ISSUE THREAD

**Issue by sammo3182** [sammo3182/interplot#39]
From Guan Wang by email

I ran the example based on the mtcars dataset 
provided in the R file ([https://cran.r-project.org/web/packages/interplot/vignettes/interplot-vignette.html](https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2Finterplot%2Fvignettes%2Finterplot-vignette.html&data=05%7C01%7C%7Cb678824568ba4a9c749608db81a13fc4%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638246301787542325%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UcOmfIAtTkGbm8gDY%2BjqDgPLeB1qsAYhtkPpr29XE9w%3D&reserved=0)). The two variables in the interaction term are "wt (continuous)" and "vs (binary)". Below is the code: 

```
library(interplot)

data(mtcars)

class(mtcars$vs);table(mtcars$vs)

m_cyl <- glm(am ~ wt * vs, data = mtcars,family = binomial(link = "logit"))
summary(m_cyl)

windows()
plot_1 <- interplot(m_cyl, var1 = "wt",var2 = "vs",
                       predPro = TRUE,
                       var2_vals = c(0,1))
plot_1

windows()
plot_2 <- interplot(m_cyl, var1 = "vs",var2 = "wt",
                    predPro = TRUE,
                    var2_vals = c(min(mtcars$wt),max(mtcars$wt)))
plot_2### 
```

**Comment by sammo3182** [comment]
Fred @fsolt , do you mind have a check of this bug? 

I guess the problem relates to the calculation of the intervals, but I can't figure out how to fix it: 

https://github.com/sammo3182/interplot/blob/74e77abd84c6b8bbb38ae760b904f8a1df3df70e/R/Interplot_default.R#L487-L508

**Comment by fsolt** [comment]
Is this issue closed then?

**Comment by sammo3182** [comment]
> Is this issue closed then?

Nope. The problem is still there. I just fixed the visualization part.
